Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -723,6 +723,7 @@ export function extend(publicAPI, model, initialValues = {}) {
 
   // Create get-set macros
   macro.setGet(publicAPI, model, ['rotationFactor']);
+  macro.get(publicAPI, model, ['keyboardManipulators']);
```

Comment:
add the full set `['mouseManipulators', 'keyboardManipulators', 'vrManipulators', 'gestureManipulators']`

Reply:
@jourdain Done.